examples/asset/asset_decompression.rs

```diff
@@ -111,7 +111,7 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             compressed: asset_server.load("data/compressed_image.png.gz"),
```

Comment:
Not sure how to handle this generically now.

Reply:
I added Handle<A>: Into<C> trait bounds. That means C needs to implement From<Handle<A>, which is true for Sprite (and Mesh as well).
May not be the best solution.